gulpfile.js

```diff
@@ -293,16 +149,21 @@ var buildExamples = function (destPath = DEST_DIR, sdkUrl = null) {
 
 // Tasks
 gulp.task('default', function (done) {
-    runSequence('clean', ['build', 'doc'], done);
+    runSequence('clean', ['build', 'docs'], done);
 });
 
 gulp.task('clean', function () {
     return del([DEST_DIR]);
 });
 
-gulp.task('build', build);
+gulp.task('docs', () => {
```

Comment:
This name change would change contributing.md.  But, we're dumping gulp soon enough and you've got it removed in a later commit, so no need to make that change here.